tests/funcalign/test_srm.py

```diff
@@ -62,6 +63,9 @@ def test_can_instantiate():
 
     # Check that runs with 2 subject
     s.fit(X)
+    sr_v0_4 = np.load('sr_v0_4.npy')
```

Comment:
Add this to imports:
`from pathlib import Path`

Then:
`Path(__file__).parent / "sr_v0_4.npy"`